bot/bot.go

```diff
@@ -178,7 +178,9 @@ func (me *Bot) welcomeMessage(config config.Config) string {
 		msg += fmt.Sprintf("> *Current price:* %.5f\n", currentPrice)
 		msg += fmt.Sprintf("> *Buy price:* %.5f\n", trade.BuyPrice)
 		msg += fmt.Sprintf("> *Sell Price:* %.5f\n", trade.SellPrice)
-		msg += fmt.Sprintf("> *Profit:* %.2f%%\n\n", (trade.SellPrice-trade.BuyPrice)/(trade.SellPrice)*100)
+		msg += fmt.Sprintf("> *Fee rate:* %.4f%%\n", binance.Fee(trade.BuyWith()))
```

Comment:
In this way the fee trading a BTC, for example, will be 0.1%, but it is not true.
When you trading any symbol you can choose pay the fees with BNB(0.075%) or not(0.1%).
So, we can't pass `trade.BuyWith()` as parameter here.
We can receive a new parameter in yaml config, asking for the fee way.